### DISCUSSION TITLE
How sweep coins from all child addresses to one main?

### DISCUSSION THREAD

**Discussion by lucker28** [1200wd/bitcoinlib#418] [Q&A] [answered]
I have many child addresses with 1-2$, I wanna sweep them to one address, but when I try to do this, it gave me txid which not pushed in blockchain. I think balance in this wallets so small, so they can't cover fee for the transfer. Any way to locally sweep all child addresses coins to 1 main address?

**Comment by mccwdev** [comment, marked as answer]
There is not such thing a local transaction. A transaction is broadcasted to the network and then will be added to the global blockchain if accepted.

But fees are pretty low at the moment. You can use the fee_per_kb parameter to specify a very low fee, and see if it is accepted and included in the blockchain.
